site2/docs/reference-metrics.md

```diff
@@ -495,6 +496,25 @@ All the jetty metrics are labelled with the following labels:
 | jetty_stats_seconds | Gauge | Time in seconds stats have been collected for. |
 | jetty_responses_bytes_total | Counter | Total number of bytes across all responses. |
 
+### Schema metrics
+
+> For a functions-worker running separately from brokers, its schema metrics are only exposed when `includeStandardPrometheusMetrics` is set to `true`.
+
+All the schema metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you have configured in the `broker.conf` file.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_schema_del_ops_failed_count | Counter | Number of delete schema failed operations. |
```

Comment:
```suggestion
| pulsar_schema_del_ops_failed_count | Counter | Number of operations that delete schemas |
```
do you intend to mean this？
if so, could you please help update other occurencies? thanks